src/components/action-panels/Withdraw.js

```diff
@@ -46,8 +46,13 @@ export default class Withdraw extends Component {
     };
 
     setMax = () => {
-        const {userInfo} = this.props
-        const val = (Math.floor(userInfo.bCdpInfo.ethDeposit * 1000000)/1000000).toString();
+        const {userInfo, currency} = this.props
+        let val
+        if(currency === 'ETH'){
```

Comment:
why don't you just use a single  XDeposit variable and avoid all these `if` statements?